### PR TITLE
Fix document

### PR DESCRIPTION
**Pull request by h-east**
Hi Bram and all,

Check it out please.

BTW,
I'm not sure about the help link policy.

What is the difference between Backtick (`) link and Bar (|) link.

For backtick link, omit a colon?
NG? \`:setlocal\`
OK? \`setlocal\`

--
Best regards,
Hirohito Higashi (h_east)

### CODE REVIEW

File: runtime/doc/channel.txt [hunk 1]
```diff
@@ -1554,36 +1554,5 @@ The "params" field is optional: >
 	"params": <list|dict>
     }
 
-Depending on the use case, you can use the ch_evalexpr(), ch_sendexpr() and
```
Comment (yegappan) [inline runtime/doc/channel.txt:1557]: Why is this text (sample LSP messages) removed?

Comment (h-east) [reply runtime/doc/channel.txt:1557]: It's a duplicate.
L1525-L1555 and L1557-L1587 are the same thing.

**Comment by brammool** [comment]
Thank you, that is a long list of improvements.

**Comment by h-east** [comment]
Hi Bram,

From https://github.com/vim/vim/pull/10620#issue-1284741832
> BTW,
> I'm not sure about the help link policy.
> 
> What is the difference between Backtick (`) link and Bar (|) link.
> 
> For backtick link, omit a colon?
> NG? `:setlocal`
> OK? `setlocal`

I apologize for asking this question almost half a year ago, but could you please answer it?
I'm still not sure what the policy is for this, so I'd appreciate it if you could let me know.

--
Best regards,
Hirohito Higashi (h_east)

**Comment by brammool** [comment]
> >From https://github.com/vim/vim/pull/10620#issue-1284741832
> > BTW,
> > I'm not sure about the help link policy.
> > 
> > What is the difference between Backtick (`) link and Bar (|) link.
> 
> I apologize for asking this question almost half a year ago, but could
> you please answer it?
> I'm still not sure what the policy is for this, so I'd appreciate it
> if you could let me know.

The backticks are for commands, they will be highlighted as such.
If it is just one command then it also works as a tag.

> > For backtick link, omit a colon?
> > NG? `:setlocal`
> > OK? `setlocal`

Depends on the context.  If it's clear that an Ex command is typed then
the colon is often left out.  In between Normal mode commands it's
better to show the colon, to make clear it's a different kind of
command.

-- 
hundred-and-one symptoms of being an internet addict:
183. You move your coffeemaker next to your computer.

 /// Bram Moolenaar -- ***@***.*** -- http://www.Moolenaar.net   \\\
///                                                                      \\\
\\\        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by h-east** [comment]
I see. Thank you.

**Pull request closed without merging**